src/perftest_parameters.c

```diff
@@ -471,8 +471,8 @@ static void usage(const char *argv0, VerbType verb, TestType tst, int connection
 		printf(" Set the Traffic Class in GRH (if GRH is in use)\n");
 
 		#ifdef HAVE_CUDA
-		printf("      --use_cuda ");
-		printf(" Use CUDA lib for GPU-Direct testing.\n");
+		printf("      --use_cuda=<cuda device id>");
```

Comment:
this part of the patch changes the syntax of the --use_cuda option while is not documented in the commit description.

Reply:
also, it will help to make it an optional argument and pick device zero if none is specified. There are quite a few places where this test is used in automation. Requiring an argument will force changes everywhere.

Reply:
We will try to figure out the best approach here, thanks for your feedback